cloudformation.properties

```diff
@@ -2,15 +2,21 @@ name=Cloudformation
 category=External Analysers
 descripton=Cloudformation rules for json and yaml, integrates cfn-nag reports.
 homepageUrl=https://github.com/Hack23/sonar-cloudformation-plugin
-archivedVersions=2.0.1,2.0.2,2.0.3,2.0.4
-publicVersions=2.0.5
+archivedVersions=2.0.1,2.0.2,2.0.3,2.0.4,2.0.5
+publicVersions=2.0.7
 
 defaults.mavenGroupId=com.hack23.sonar
 defaults.mavenArtifactId=sonar-cloudformation-plugin
 
+2.0.7.description=Bugfixes, updated rule descriptions and nist 800-53 tags.
+2.0.7.date=2020-10-31
+2.0.7.sqVersions=[7.9,LATEST]
+2.0.7.changelogUrl=https://github.com/Hack23/sonar-cloudformation-plugin/releases/tag/sonar-cloudformation-plugin-2.0.7
+2.0.7.downloadUrl=https://oss.sonatype.org/service/local/repositories/releases/content/com/hack23/sonar/sonar-cloudformation-plugin/2.0.7/sonar-cloudformation-plugin-2.0.7.jar
+
 2.0.5.description=Support 6 new cfn-nag rules.
 2.0.5.date=2020-04-29
-2.0.5.sqVersions=[7.9,LATEST]
+2.0.5.sqVersions=[7.9,8.4]
```

Comment:
You need to update this end range to 8.5.*. Otherwise users of the current SQ version won't be prompted to upgrade to your new version in-app.